code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -661,3 +661,61 @@
 				w_uniform.add_fingerprint(M)
 
 			..()
+
+
+/mob/living/carbon/human/damage_clothes(damage_amount, damage_type = BRUTE, damage_flag = 0, def_zone)

```

Comment:
it's less beautiful than the older version, it's now pretty much 60% copypasted from acid_act(), but it prevents damaging both the suit and jumpsuit at the same time.